src/features/alias/helpers/setAlias.ts

```diff
@@ -41,10 +41,12 @@ export default function setAlias({
                 try {
                     yield destroy(owner, address)
                 } catch (e) {
-                    yield toast({
-                        title: i18n('aliasToast.failedToDestroy'),
-                        type: ToastType.Error,
-                    })
+                    yield put(
```

Comment:
`yield toast(…)` gets replaced with `yield put(ToasterAction.show(…))`. It's consistent now. This is the way to display a normal toast from both any saga and from anywhere in the UI (i.e. via `dispatch(ToasterAction.show(…))`).